### PR TITLE
feat(Readme): Added VS Code Marketplace Ratings badge

### PR DESCRIPTION
**Pull request by andregallacci**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Color Vision
 
+![](https://img.shields.io/visual-studio-marketplace/r/wilsonsio.color-vision)
```
Comment (SgiobairOg) [inline README.md]:
```suggestion
![Visual Studio Marketplace Rating](https://img.shields.io/visual-studio-marketplace/r/wilsonsio.color-vision)
```

Comment (SgiobairOg) [reply README.md]: Just the one addition, thanks for the contribution!

**Comment by andregallacci** [comment]
@SgiobairOg Thanks for the suggestion!

**Review by SgiobairOg — APPROVED** [review]
All good, thanks!

**Pull request merged**